README.md

```diff
@@ -216,9 +216,39 @@ onediffusion build stable-diffusion
 onediffusion build stable-diffusion-xl
 ```
 
-To specify the model to be packaged into the Bento, use `--model-id`. Otherwise, OneDiffusion packages the default model into the Bento. If the model does not exist locally, OneDiffusion downloads the model automatically.
+To specify the model to be packaged into the Bento, use `--model-id`. Otherwise, OneDiffusion packages the default model into the Bento. If the model does not exist locally, OneDiffusion downloads the model automatically. In addition, the pipeline to use can also be specified through `--pipeline`. By default, OneDiffusion uses the `text2image` pipeline.
 
-Once your Bento is ready, you can push it to [BentoCloud](https://www.bentoml.com/cloud).
+To package LoRA weights into the Bento, use the `--lora-dir` option to specify the directory where LoRA files are stored. These files can be dynamically loaded to the model when deployed with Docker or BentoCloud to create task-specific images.
+
+```bash
+onediffusion build stable-diffusion-xl --lora-dir "/path/to/lorafiles/dir/"
+```
+
+If you only have a single LoRA file to use, run the following instead:
+
+```bash
+onediffusion build stable-diffusion-xl --lora-weights "/path/to/lorafile"
+```
+
+Each Bento has a randomly created `BENTO_TAG` as its unique identifier. To customize it, use `--name` and `--version` options.
```

Comment:
The bento tag's name is construct from model_name, model_id and other factors. the tag's version is also not randomly generated but match model's version. The model's tag is followed huggingface's commit hash. That's why the same command will generally generate the same tag. Hence I think here we should not use "randomly". Can we say that by default each bento will generate a name containing information like model name, model_id etc.?

Reply:
Thanks. Changed.